theme_report/generate_theme_report.py

```diff
@@ -0,0 +1,103 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait # available since 2.4.0
+from selenium.webdriver.support import expected_conditions as EC # available since 2.26.0
```

Comment:
Is Selenium required because the program uses it to authenticate with username and password?

You've already looked for an API-based solution for this?

Reply:
Per our discussion yesterday, there is no proper API for getting the theme info. That's why Selenium is used here to pass the auth page and retrieve subaccount theme settings.

Reply:
Makes sense.  It's too bad that Canvas doesn't have an API function for accessing themes.  As you said, maybe after this is complete and working, you can share it with the Canvas Community forum and ask whether anyone knows of a better way to do it.